test/new/db/tools/rabin2

```diff
@@ -104,6 +104,14 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=rabin2 -SSS
+FILE=../bins/elf/analysis/x86-helloworld-gcc
+CMDS=<<EOF
+!rabin2 -SSS ${R2_FILE}|wc -l|awk "{print \$1}"
+EOF
+EXPECT='13'
```

Comment:
```suggestion
EXPECT=13
```

The `..` syntax is deprecated and might not work in the future.

Reply:
Any suggestion on how to fix '..' syntax?
Besides that, I saw other test cases used it as well